ledger-tool/src/main.rs

```diff
@@ -103,8 +105,37 @@ fn main() {
         ))
         .subcommand(SubCommand::with_name("bounds").about("Print lowest and highest non-empty slots. Note: This ignores gaps in slots"))
         .subcommand(SubCommand::with_name("json").about("Print the ledger in JSON format").arg(&starting_slot_arg))
-        .subcommand(SubCommand::with_name("verify").about("Verify the ledger's PoH"))
-        .subcommand(SubCommand::with_name("prune").about("Prune the ledger at the block height").arg(
+        .subcommand(
+            SubCommand::with_name("verify")
+            .about("Verify the ledger's PoH")
+            .arg(
+                Arg::with_name("no_snapshot")
+                    .long("no-snapshot")
+                    .takes_value(false)
+                    .help("Do not start from a local snapshot if present"),
+            )
+            .arg(
+                Arg::with_name("account_paths")
+                    .long("accounts")
+                    .value_name("PATHS")
+                    .takes_value(true)
+                    .help("Comma separated persistent accounts location"),
+            )
+        .arg(
+            Arg::with_name("halt_at_slot")
+                .long("halt-at-slot")
+                .value_name("SLOT")
+                .takes_value(true)
+                .help("Halt processing at the given slot"),
+        )
+        .arg(
+            clap::Arg::with_name("skip_ledger_poh_verify")
+                .long("skip-ledger-poh-verify")
```

Comment:
This is hard to read. How about replacing all instances of `ledger_poh_verify` with just `poh_verify` which is fairly consistent with `sig_verify`?

Reply:
✔️

Reply:
Hyphenated version too!